index.qmd

```diff
@@ -53,127 +53,135 @@ bibliography: references.bib
+We refer to these networks as species agnostic as the determination of links between species is not due to the properties of the node but rather due to the property of the *expected* structure of the network. This means these networks are usually not species specific and have a limited use in real-world context. Finally, although these models are data light (often requiring only species richness and the number of expected links) there are assumptions that need to be made regarding what the expectations are for network structure. There are two sub-categories of species agnostic networks:
 
-These models are data light but there are some decisions that need to be made regarding what the expectations are on network structure.
+Random network models [*e.g.,* @erdosRandomGraphs1959; @fortunaHabitatLossStructure2006; @bascompteNestedAssemblyPlantanimal2003] represent a 'process free' model. These models are not explicitly tied to a process discussed in @sec-process, rather links are randomly distributed among nodes. Although these models lack real world tractability [@bascompteNetworksEcology2007] they are often used as a 'null hypothesis' to ask questions about network structure [*e.g.,* @banvilleWhatConstrainsFood2023; @strydomSVDEntropyReveals2021].
+
+Stochastic network models use a probabilistic rule-set about diet choice and niche breadth to reflect fundamental ideas of foraging biology. **RATHER THAN ALLESINA, I'D SUGGEST PRESENTING THE 5 RULES IN THE ADBM EXPLANATION OF THE NICHE MODEL** These models that are based on the compartmentation and acquisition of energy for species at different trophic levels [@allesinaFoodWebModels2009; @krauseCompartmentsRevealedFoodweb2003] and that network structure can be determined by distributing interactions along single dimension [the 'niche axis'; @allesinaGeneralModelFood2008]. Typically these models parametrise some aspect of the network structure [although see @allesinaFoodWebModels2009 for a parameter-free model]. These models include the most commonly used network generator, the Niche model [@williamsSimpleRulesYield2000], as well as the original Cascade model [@cohenCommunityFoodWebs1990] and the derived Nested hierarchy model [@cattinPhylogeneticConstraintsAdaptation2004]. These models often form the basis for dynamic models *e.g.,* the allometric trophic network [@broseAllometricScalingEnhances2006] and bioenergetic food web models [@delmasSimulationsBiomassDynamics2017].
```

Comment:
I don't quite understand the need to go all out with describing the niche model? Its just going to bloat up the text even more and we don't go into any real detail with the other models